src/api/tokenList/TokenListApi.ts

```diff
@@ -59,6 +59,20 @@ export class TokenListApiImpl extends GenericSubscriptions<TokenDetails[]> imple
     return this._tokensByNetwork[networkId] || []
   }
 
+
+  private mergeTokenLists(baseList: TokenDetails[], newList: TokenDetails[]): TokenDetails[] {
+    const seenAddresses = new Set<string>()
+    const result: TokenDetails[] = []
+
+    baseList.concat(newList).forEach(token => {
+      if (!seenAddresses.has(token.address.toLocaleLowerCase())) {
+        seenAddresses.add(token.address.toLocaleLowerCase())
```

Comment:
Why `Locale`? No need, `toLowerCase` is enough

Reply:
Oops, auto-complete late in the evening sleepy coding. Thanks for catching it :)